src/2023/styles/styles.css

```diff
@@ -240,6 +240,17 @@ a {
   font-weight: bold;
 }
 
+.call-for-speakers-ended {
+  margin: 1rem;
+  font-size: var(--step--1);
+  color: var(--gray);
+  opacity: 0.8;
```

Comment:
Can you check the contrast?

This looks the same as the mailing list info, which itself has low contrast as reported by Lighthouse.

Reply:
Contrast is 4.08 which is AA+, but lighthouse wants 4.5 if I remember correctly.

![Screenshot 2023-04-19 at 08 19 07](https://user-images.githubusercontent.com/1409672/232983882-be077295-fc2a-44e9-8000-6331db177a97.png)

Reply:
Hmm, it could be a difference in rendering between fonts and operating systems. My Firefox on Linux says the contrast is 4.03.

Alternatively, we could put the message black on white below the hero section.

Reply:
I pushed a new version, that looks like this:

![Screenshot 2023-04-19 at 09-56-41 Helvetic Ruby Conference 2023](https://user-images.githubusercontent.com/1409672/233009781-d29a4bdd-9235-4f74-879c-a9d8d4f73af6.png)

Reply:
Both are fine for me. I suppose we'll remove the message anyhow once we start listing speakers.

Reply:
@crackofdusk 👍 I agree! I like the Gold version as it is more obvious.